packages/react-reconciler/src/ReactProfileTimer.js

```diff
@@ -0,0 +1,130 @@
+/**
+ * Copyright (c) 2013-present, Facebook, Inc.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ * @flow
+ */
+
+import type {Fiber} from './ReactFiber';
+import type {StackCursor} from './ReactFiberStack';
+
+import warning from 'fbjs/lib/warning';
+import ReactFiberStack from './ReactFiberStack';
+
+const hasNativePerformanceNow =
+  typeof performance === 'object' && typeof performance.now === 'function';
+
+let now;
+if (hasNativePerformanceNow) {
+  now = function() {
+    return performance.now();
+  };
+} else {
+  now = function() {
+    return Date.now();
+  };
+}
```

Comment:
I used the host config function originally but removed it because my understanding was that we planned to change that in the future in a way that would not be compatible with this use case. cc @acdlite

Reply:
If this is not the case, I can change it back to using the host config function. I'll have to make a change two test renderer to support mocking that function but that should be easy.

Reply:
@bvaughn I miscommunicated. I thought you were talking about the scheduler’s `recalculateCurrentTime` function. `HostConfig.now` is fine to use. My bad!